components/cms-services/pkg/endpoint/asyncapi/v1/conversion_endpoint.go

```diff
@@ -22,6 +22,7 @@ var _ endpoint.Mutator = Convert(nil)
 
 // Mutate converts the AsyncAPI specification from version 1.* to version 2.0.0-rc1.
 func (c Convert) Mutate(ctx context.Context, reader io.Reader, metadata string) ([]byte, bool, error) {
+	//add metrics here?
```

Comment:
Yes :P

Reply:
No, it is already added in `components/cms-services/pkg/runtime/endpoint/mutation_endpoint.go`, so thanks to that it is generic.